### PR TITLE
Chore: Add fixes to datagrid examples and improve rendering

### PR DESCRIPTION
**Pull request by tmcconechy**
**Explain the details for making this change. What existing problem does the pull request solve?**
Working with @deep7102 we found two pages not working so I fixed them. In addition in this pr there is:

- worked on checkbox TODOs
- improved datagrid rendering speed with a flag
- fixes the single select and virtual scroll examples
- add a test
- some minor cleanup (spaces in the html properties and wierd named event)

**Related github/jira issue (required)**:
N/A

**Steps necessary to review your pull request (required)**:
- go to http://localhost:4300/ids-virtual-scroll/ and it should work as before (minor style fixes)
- go to http://localhost:4300/ids-data-grid/virtual-scroll.html and it should work now (compare to https://main.wc.design.infor.com/ids-data-grid/virtual-scroll.html)
- go to http://localhost:4300/ids-data-grid/single-selection.html and it should work now (compare to https://main.wc.design.infor.com/ids-data-grid/single-selection.html)

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.

### CODE REVIEW

File: src/components/ids-checkbox/README.md [hunk 1]
```diff
@@ -106,6 +106,15 @@ Hide label and show only checkbox:
...
 - `value` {string} set the checkbox value.
 
+## Events
+
+- `change` The change event is fired when a checkbox element's value is committed by the user. Unlike the input event, the change event is not necessarily fired for each alteration to an element's value.
+- `input` The input event fires when the value of a checkbox element has been changed.
+
+## Methods
+
+- `dismiss` Removes the tag from the page.
```
Comment (deep7102) [inline src/components/ids-checkbox/README.md:116]: Is this a typo? not sure about this method `dismiss` in checkbox

**Review by deep7102 — COMMENTED** [review]
Looks all good!
only see the radio style which might not belong to this PR and can handle as separate issue.
http://localhost:4300/ids-data-grid/single-selection.html

**Review by jmacaluso711 — DISMISSED** [review]

**Comment by tmcconechy** [comment]
I fixed the radio styles @deep7102

**Review by deep7102 — APPROVED** [review]
Thanks! looks all good 🍌

**Pull request merged**